egs/apiai_decode/s5/README

```diff
@@ -0,0 +1,33 @@
+Example scripts on how to use a pre-trained chain model and kaldi base code to recognize any number of wav files.
```

Comment:
mention the language (English), and say what type of data it is optimized for if you can.